<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FindDialog</class>
 <widget class="QDialog" name="FindDialog">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>450</width>
    <height>140</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>查找</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <widget class="QLabel" name="findLabel">
     <property name="text">
      <string>查找目标:</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <widget class="QLineEdit" name="findEdit"/>
   </item>
   <item row="0" column="2">
    <widget class="QPushButton" name="findButton">
     <property name="text">
      <string>查找下一个(&amp;F)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="0">
    <widget class="QCheckBox" name="matchCaseCheck">
     <property name="text">
      <string>区分大小写(&amp;C)</string>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QGroupBox" name="directionGroup">
     <property name="title">
      <string>方向</string>
     </property>
     <layout class="QHBoxLayout" name="directionLayout">
      <item>
       <widget class="QRadioButton" name="upwardBtn">
        <property name="text">
         <string>向上(&amp;U)</string>
        </property>
       </widget>
      </item>
      <item>
       <widget class="QRadioButton" name="downwardBtn">
        <property name="text">
         <string>向下(&amp;D)</string>
        </property>
        <property name="checked">
         <bool>true</bool>
        </property>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
   <item row="1" column="2">
    <widget class="QPushButton" name="cancelButton">
     <property name="text">
      <string>取消</string>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
